web/ui/templates/status.html

```diff
@@ -54,7 +54,8 @@ <h2 id="alertmanagers">Alertmanagers</h2>
         </tr>
         {{range .Alertmanagers}}
         <tr>
-          <td>{{.}}</td>
+          {{/* Alertmanager URLs always have Scheme, Host and Path set */}}
+          <td>{{.Scheme}}://<a href="http://{{.Host}}">{{.Host}}</a>{{.Path}}</td>
```

Comment:
I could make it use `{{.Scheme}}` inside the link too, but figured if it's truly non-http then the text should probably reflect reality but the link should be http to be clickable with a web browser.